### PR TITLE
Added `/constants` endpoint

### PR DESCRIPTION
**Pull request by builder-247**

### CODE REVIEW

File: routes/spec.js [hunk 1]
```diff
@@ -4343,6 +4343,28 @@ You can find data that can be used to convert hero and ability IDs and other inf
...
+          required: true,
+          type: 'string',
+        }],
+        route: () => '/constants/:resource',
+        func: (req, res, cb) => {
+          const { resource } = req.params;
+          if (resource in constants) {
+            return res.json(constants[resource]);
+          }
+          return cb();
```
Comment (howardchung) [inline routes/spec.js]: does this go to the 404 handler?

File: routes/spec.js [hunk 2]
```diff
@@ -4343,6 +4343,28 @@ You can find data that can be used to convert hero and ability IDs and other inf
         },
       },
     },
+    '/constants/{resource}': {
```
Comment (howardchung) [inline routes/spec.js]: My worry with this name is that people won't realize this means static game data like hero stats etc.

Maybe we can add some keywords

Comment (builder-247) [reply routes/spec.js]: Any ideas?

File: routes/spec.js [hunk 3]
```diff
@@ -4343,6 +4343,28 @@ You can find data that can be used to convert hero and ability IDs and other inf
...
+      get: {
+        summary: 'GET /constants',
+        description: 'Get constant data mirrored from the dotaconstants repository',
```
Comment (howardchung) [inline routes/spec.js]: maybe "static game data" instead of constant data

**Comment by howardchung** [comment]
Tests failing? My guess is it's trying and failing to test the new endpoint.

**Comment by builder-247** [comment]
All  builds seem to be failing for some reason during setup

**Pull request merged**